packages/web-config-server/src/apiV1/measureData.js

```diff
@@ -257,7 +263,7 @@ export default class extends DataAggregatingRouteHandler {
       ...restOfPresentationConfig
     } = config;
 
-    const { dataSourceType = DATA_SOURCE_TYPES.SINGLE, periodGranularity } =
+    const { dataSourceType = DATA_SOURCE_TYPES.CUSTOM, periodGranularity } =
```

Comment:
What's the `CUSTOM` about?

Reply:
Oh good catch. This is because when `dataSourceType` is `single` the server will attempt to fetch metadata through the aggregator (Fetching from DHIS2 according to the comments) Since new style reports don't have a `measureBuilderConfig` they always take the default `dataSourceType` and error during that process, so I swapped it to `custom` which is more accurate for them.

This line is still wrong though because it will break the actual `single` type overlays which assume this default is in place so don't define `dataSourceType` in their configs. We could add a migration to add that `dataSourceType` to the configs of those and keep this defaulting to `custom`

Alternatively we could just make the default conditional `dataSourceType = legacy ? DATA_SOURCE_TYPES.SINGLE : DATA_SOURCE_TYPES.CUSTOM`

Let me know if you have any preference between those. I think the first solution is probably more correct.

Reply:
Ah cool. I actually think we should go for the second solution, because that maintains status quo for devs who are familiar with developing legacy dashboard reports, which will probably continue for some time until the report server completely catches up.

It also doesn't look like you've added the `CUSTOM` type to the definition?

Reply:
Oh wow I had just assumed it was already defined and ran with it. Good catch.